startup/interface-rename-sideway.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Prepare for interface rename
+Before=multi-user.target
+Before=graphical.target
+Before=network-online.target
```

Comment:
Isn't `network-online.target` too late?

I would expect this to happen before any network interfaces are configured. So probably `Before=network-pre.target`

There is more information about this here: https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/

Reply:
This one also from initscript converted service, 
`network-pre` sounds better.